src/theme/elevation.ts

```diff
@@ -6,10 +6,10 @@ export type Elevation = {
 };
 
 const elevation: Elevation = {
-  '01': '0px 2px 4px 0px rgba(0, 0, 0, 0.15)',
```

Comment:
just a little tip here, i removed from the shadow this metric as it will still work and it will also serve the same for other purposes like filter